content/3_Silver/Binary_Search_Ans.mdx

```diff
@@ -362,6 +413,21 @@ static long search(){
 
 </JavaSection>
 
+<PySection>
+
+```python
+def search():
+    pos = 0
+	a = mx = 2E9
+	while a >= 1:
```

Comment:
indentation looks a little inconsistent here, and this matters for python.

Reply:
Why... the indentation is correct in my vscode
![image](https://user-images.githubusercontent.com/49191786/99027383-0a271900-25a8-11eb-8d98-b3be81b41812.png)

Reply:
be careful with tabs vs spaces.... https://usaco.guide/liveupdate/

Reply:
I see